release_bot/cli.py

```diff
@@ -31,6 +31,9 @@ def parse_arguments():
                             default='')
         parser.add_argument("-v", "--version", help="display program version", action='version',
                             version=f"%(prog)s {configuration.version}")
+        parser.add_argument("-n", "--dry_run", default = False, 
```

Comment:
I think the options parameter could (and should) be called `dry-run` with a dash. You can specify `dest="dry_run"` as `add_argument` parameter to avoid invalid variable name. Or see another way how to do it [here](https://ironboundsoftware.com/blog/2018/01/20/python-argparse-arguments-with-dashes/)

Reply:
Looks like there is no need for a `metavar`. I found out that Argparse automatically converts a dash to an underscore. So, `dry-run` in the arguments can be used as `dry_run` in the program.